### PR TITLE
Update docs to reflect field paths separated by '.'

### PR DESCRIPTION
**Pull request by slightfoot**

### CODE REVIEW

File: packages/cloud_firestore/lib/src/query.dart [hunk 1]
```diff
@@ -97,7 +97,11 @@ class Query {
       new CollectionReference._(firestore, _pathComponents);
 
   /// Creates and returns a new [Query] with additional filter on specified
-  /// [field].
+  /// [field]. [field] refers to a field in a document.
+  ///
+  /// The [field] may consist of a single field name (referring to a top level
+  /// field in the document), or a series of field names seperated by
+  /// forward-slashes '.' (referring to a nested field in the document).
```
Comment (kroikie) [inline packages/cloud_firestore/lib/src/query.dart]: I assume you mean `separated by dots`?

**Comment by slightfoot** [comment]
Thanks @kroikie that was embarrassing. Correction commited. Just waiting on the checks.

File: packages/cloud_firestore/lib/src/query.dart (hunk 1 above)
Comment (slightfoot) [reply packages/cloud_firestore/lib/src/query.dart]: Corrected.

**Comment by kroikie** [comment]
@slightfoot Thanks for this PR, could you also bump the version number then update the CHANGELOG so I can publish it and get the updated docs into our generated documentation.

**Comment by slightfoot** [comment]
@kroikie version bumped to v0.7.4 and CHANGELOG updated. Just waiting on checks. Cheers.

**Comment by Hixie** [comment]
@kroikie Can you see why the tests are failing?

**Comment by kroikie** [comment]
@slightfoot would you please rebase this PR, tests should pass then.

**Comment by kroikie** [comment]
@slightfoot sorry there seems to still be some formatting issues. Could you resolve those then I'll merge?

**Comment by slightfoot** [comment]
@kroikie The formatting changes that are an issue are not from my commits. Someone else has merged without dartfmt on master before my commits. What do you want to do?
```
packages/google_maps_flutter/example/lib/map_ui.dart
packages/google_sign_in/lib/widgets.dart
packages/path_provider/example/lib/main.dart
packages/shared_preferences/example/lib/main.dart
```

**Comment by kroikie** [comment]
@slightfoot You are correct, the formatting issues are not on your end at all. Once you rebase those changes should no longer be part of your PR.

**Comment by slightfoot** [comment]
@kroikie rebased.

**Pull request merged**